extra/3-magic-8-ball.js

```diff
@@ -101,7 +126,7 @@ test("magic 8 ball returns different values each time", () => {
     );
   }
 
-  let seenPositivities = new Set(Array.from(answers.values()).map(checkAnswer));
+  let seenPositivities = new Set(Array.from(seenAnswers.values()).map(checkAnswer));
```

Comment:
Arghhh...

Not your code, but I hate it. No one uses `new Set` in the industry, this isn't super easy to read either

Reply:
That worried for a minute as I was certain I hadn't written this